prow/flagutil/k8s_client.go

```diff
@@ -61,12 +60,11 @@ func (o *KubernetesClientOptions) Validate(dryRun bool) error {
 	return nil
 }
 
-// Client returns a Kubernetes client.
+// KubeClient returns a Kubernetes client.
 func (o *KubernetesClientOptions) KubeClient() (kubernetes.Interface, error) {
 	return kube.GetKubernetesClient(o.masterURL, o.kubeConfig)
 }
 
-// Client returns a Kubernetes client.
 func (o *KubernetesClientOptions) ProwJobClient() (versioned.Interface, error) {
```

Comment:
missing a comment here?

Reply:
The old comment was wrong so I removed it until I add another one. It's updated now.